includes/phing/build/help/drush.xml

```diff
@@ -414,6 +442,25 @@
 
     <target name="drush-gdpr-dump" hidden="true" description="Make a sanitized dump of database.">
         <propertyregex property="profile.core.major" subject="${profile.core}" pattern="^([0-9]+)\." match="$1" override="true" />
+
+        <echo message="Backing up database ${db.name} to ${database-file}." />
+        <trycatch>
+            <try>
+                <exec command="${toolkit.dir.bin.drush} -r ${build.platform.dir} gdpr-sql-dump --help" checkreturn="true" />
+            </try>
+            <catch>
+                <drush command="dl" assume="yes" root="${build.platform.dir}" alias="@${drush.alias.default}" bin="${toolkit.dir.bin.drush}" verbose="${drush.verbose}" color="${drush.color}">
+                    <param>gdpr-1.0-alpha11 checklistapi-1.2</param>
```

Comment:
Move modules into prop, we locked it to alpha11 because alpha12 removed support to php 5.6.